cms/djangoapps/contentstore/views/organization.py

```diff
@@ -18,6 +17,7 @@ class OrganizationListView(View):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         """Returns organization list as json."""
-        organizations = get_organizations()
-        org_names_list = [(org["short_name"]) for org in organizations]
+        organizations = get_enabled_organization(request)
+        org_names_list = [(org['short_name']) for org in organizations]
```

Comment:
nit: @burhandodhy we have only one line change here.
`organizations = get_enabled_organization(request)`
Please remove other changes